dev-tools/cmd/kibana_index_pattern/kibana_index_pattern.go

```diff
@@ -32,18 +32,21 @@ func main() {
 		os.Exit(1)
 	}
 
-	indexPatternGenerator, err := kibana.NewGenerator(*index, *beatName, *beatDir, *version)
-	if err != nil {
-		fmt.Fprintf(os.Stderr, err.Error())
-		os.Exit(1)
-	}
-
-	pattern, err := indexPatternGenerator.Generate()
-	if err != nil {
-		fmt.Fprintf(os.Stderr, err.Error())
-		os.Exit(1)
-	}
-	for _, p := range pattern {
-		fmt.Fprintf(os.Stdout, "-- The index pattern was created under %v\n", p)
+	version5, _ := common.NewVersion("5.0.0")
+	version6, _ := common.NewVersion("6.0.0")
```

Comment:
should this be `6.0.0` already or rather be `6.0.0-rc1`?
As you mentioned in the former PR that we should take the `meta` information of versions into account, and there are breaking changes between pre-releases, the generator now treats `6.0.0.rc1 < 6.0.0`.

Reply:
As we don't pass the version in I would stick with 6.0.0 as I don't think this is a field we should change over time with each release. Hopefully we can get rid of it completely in the long run as index pattern could be generated on request now that we have it in Golang so we would not have to pregenerate them.